tests/infrastructure/test_ialirt_processing_construct.py

```diff
@@ -8,35 +8,36 @@
 def get_nlb_dns(stack_name, port, container_name):
     """Retrieve DNS for the NLB from CloudFormation."""
     client = boto3.client("cloudformation")
-    response = client.describe_constructs(StackName=stack_name)
+    response = client.describe_stacks(StackName=stack_name)
     output_key = f"LoadBalancerDNS{container_name}{port}"
     outputs = response["Stacks"][0]["Outputs"]
     for output in outputs:
-        if output["OutputKey"] == output_key:
+        if output_key in output["OutputKey"]:
             return output["OutputValue"]
     raise ValueError(f"DNS output not found for port {port} in stack.")
 
 
 @pytest.mark.xfail(reason="Will fail unless IALiRT stack is deployed.")
```

Comment:
Is this test still xfail worthy? Or does it work now?

Reply:
It is xfail worthy since the stack has to be deployed to pass the test. It is a chicken and the egg situation.